readme.md

```diff
@@ -28,15 +28,60 @@ In order to make changes to the database schema a local Supabase instance is nee
 * Committed migrations that are merged to the main branch are automatically deployed.
 
 ### Data
-Until the topdesk api is used as primary data source, we'll read CSV from 
-`data/studieplekken-latest`. When a new version of the data comes in from TU, 
-do the following:
-
-0. convert the incoming file from *xlsx* to *csv*
-0. save to `data` directory as `studieplekken-v<version>.csv`
-0. update the symlink to point to the latest version:
-```sh
-ln -sf studieplekken-v<version>.csv data/studieplekken-latest.csv
+The Spacefinder combines data from different sources:
+
+* **Locations**: the TU Delft provides a list of locations as a CSV file (`data/studieplekken.csv`). The file is generated from a system of TU Delft Education and Student Affairs (ESA) and emailed to the Spacefinder team. Since new CSV files are rarely received, the location data is stored in the codebase and compiled into useable files (`src/data/buildings.json` and `spaces.json`) during build-time. The TU Delft plans to make the location data available through a TopDesk API in the future, but there is no timeline available.
```

Comment:
Currently `data/studieplekken.csv` is still versioned in the `data/` directory. Do we agree to remove the other versions? And should I remove those versioned files in this PR?

Reply:
Resolved by https://github.com/voorhoede/tudelft-spacefinder/pull/160/commits/86a96a6d5369d21073c1d55bd8e1a3752246b590